dotnet/src/Connectors/Connectors.Memory.Qdrant/MockableQdrantClient.cs

```diff
@@ -109,7 +109,7 @@ public virtual Task<UpdateResult> CreatePayloadIndexAsync(
         string fieldName,
         PayloadSchemaType schemaType = PayloadSchemaType.Keyword,
         CancellationToken cancellationToken = default)
-        => this._qdrantClient.CreatePayloadIndexAsync(collectionName, fieldName, schemaType, cancellationToken: cancellationToken);
+        => this._qdrantClient.CreatePayloadIndexAsync(collectionName, $"\"{fieldName}\"", schemaType, cancellationToken: cancellationToken);
```

Comment:
> BTW do we also need to go over other non-SQL connectors to make sure they're safe etc.?

@roji I went through the non-SQL connectors (except of cosmos) and this was the only line of code I needed to change to get the tests passing. We are usually using strongly typed .NET APIs for these and/or JSON serializer that does all the escaping for us.